contrib/debian/netdata.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
```

Comment:
RedHat systems don't use /etc/default, but something like /etc/sysconfig. So you'd end up with trying to substitute/remove lines from a single unit file to make it match. The dependency files are all different in .rpm systems as well (e.g. http.service rather than apache2.service), though I removed them from the Debian unit file from the above discussion.

Most other package I've seen ship different unit files for .deb vs .rpm systems for this very reason. Unless you can see a way to cleanly contain the differences. I would doubt it's worth it.